README.md

```diff
@@ -345,6 +345,13 @@ options:
       - "^Signed-off by \\s+$"
     # If true, GitHub reviews can be used for approval. Default is true.
     github_review: true
+    # Just like the "comments" and "comment_patterns" options, but used in combination
+    # with "github_review" to determine approval status. Both default to an empty list.
+    github_review_and_comments:
+      - "lgtm"
+    github_review_and_comment_patterns:
+      - "[Ll][Gg][Tt][Mm]"
```

Comment:
some feedback from @samandmoore was to change these names removing the "and" so: `github_review_comments` and `github_review_comment_patterns` to better describe what they're used for.

Reply:
yeah i like that!